server.ts

```diff
@@ -282,9 +286,12 @@ app.post("/callback/joinConference", async (req, res) => {
   );
   let conference = conferences.get(conferenceId);
   if (conference) {
-    let participantId = await bandwidthRtc.createParticipant(conferenceId);
+    let participant: [string, string] = await bandwidthRtc.createParticipant(conferenceId);
+    let participantId = participant[0];
+    let deviceToken = participant[1]
     conference.participants.set(participantId, {
       id: participantId,
+      deviceToken: deviceToken,
       status: "pending",
       name: req.body.from,
```

Comment:
Looks like we can have some code reuse here.  Same logic as above, this part where we call the sdk to create a participant and then add the participant to the conference.participants map.  suggest a addParticipant(...) private function.